src/commands/build/wranglerjs/mod.rs

```diff
@@ -17,32 +17,75 @@ use std::iter;
 use std::path::{Path, PathBuf};
 use std::process::Command;
 
-fn random_chars(n: usize) -> String {
-    let mut rng = thread_rng();
-    iter::repeat(())
-        .map(|()| rng.sample(Alphanumeric))
-        .take(n)
-        .collect()
-}
+use crate::settings::project::Project;
+
+use crate::terminal::message;
 
 // Run the underlying {wranglerjs} executable.
 //
 // In Rust we create a virtual file, pass the pass to {wranglerjs}, run the
 // executable and wait for completion. The file will receive the a serialized
 // {WranglerjsOutput} struct.
 // Note that the ability to pass a fd is platform-specific
-pub fn run_build(
-    wranglerjs_path: PathBuf,
-    wasm_pack_path: PathBuf,
-    webpack_config_path: PathBuf,
-    bundle: &Bundle,
-) -> Result<WranglerjsOutput, failure::Error> {
+pub fn run_build(cache: &Cache, project: &Project) -> Result<(), failure::Error> {
+    let (mut command, temp_file, bundle) = setup_build(cache, project)?;
+
+    info!("Running {:?}", command);
+
+    let status = command.status()?;
+    info!("it ran");
```

Comment:
I'm not sure this log is very useful?

Reply:
My bad, this is from debugging and I missed it.
```suggestion
```